### PR TITLE
[style] Removed duplicated import

### PR DESCRIPTION
**Pull request by bwagnerr**
There was a duplicate import for get_versions, that was not at
the top of the file, that caused a pep warning and was fixed
in this commit

### CODE REVIEW

File: src/leap/keymanager/__init__.py [hunk 1]
```diff
@@ -922,6 +922,5 @@ def _get_domain(url):
     """
     return urlparse(url).hostname
 
-from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
```
Comment (meskio) [inline src/leap/keymanager/__init__.py]: This code doesn't work, as in line 87 `get_versions` get deleted. I don't understand why the set of `__version__` is duplicated in this code. I'll be happy to remove all this block at the end of the file and just keep the version setting on the top.
```
Traceback (most recent call last):
  File "/home/meskio/dev/leap/env/local/lib/python2.7/site-packages/twisted/trial/runner.py", line 605, in loadByNames
    things.append(self.findByName(name))
  File "/home/meskio/dev/leap/env/local/lib/python2.7/site-packages/twisted/trial/runner.py", line 409, in findByName
    return reflect.namedAny(name)
  File "/home/meskio/dev/leap/env/local/lib/python2.7/site-packages/twisted/python/reflect.py", line 303, in namedAny
    topLevelPackage = _importAndCheckStack(trialname)
  File "/home/meskio/dev/leap/env/local/lib/python2.7/site-packages/twisted/python/reflect.py", line 242, in _importAndCheckStack
    return __import__(importName)
  File "/home/meskio/dev/leap/keymanager/src/leap/keymanager/__init__.py", line 925, in <module>
    __version__ = get_versions()['version']
exceptions.NameError: name 'get_versions' is not defined
```

Comment (kalikaneko) [inline src/leap/keymanager/__init__.py]: uh, you're right, I was too quick in merging this.
submitting fix...

Comment (kalikaneko) [inline src/leap/keymanager/__init__.py]: the duplication comes from the recent versioneer install btw.

**Comment by kalikaneko** [comment]
https://github.com/leapcode/keymanager/pull/102 fixes this

**Pull request closed without merging**